tests/generators/sanity/main.py

```diff
@@ -34,6 +34,7 @@ def cases_fn() -> Iterable[gen_typing.TestCase]:
 if __name__ == "__main__":
     phase_0_mods = {key: 'eth2spec.test.phase0.sanity.test_' + key for key in [
         'blocks',
+        'multi_operations',
```

Comment:
If we want to make `multi_operations` a new handler as the status quo, then we have to modify test format docs: https://github.com/ethereum/eth2.0-specs/tree/aab58e47008e161336dd591bd52e3c39c046a288/tests/formats/sanity

IMHO it's better to merge the `test_multi_operations.py` tests into `test_blocks.py` to share the same test format.

Reply:
yeah, that's fair. Just trying to get a clean playground to dev in. `test_blocks` is unwieldy!